### PR TITLE
Update README, bors and some .gitignore

### PR DESCRIPTION
**Pull request by charleskawczynski**
# Description

 - Adds some things to the `.gitignore`
 - Updates the README to reflect recent changes (and adds docs-build and bors badges)
 - Updates the bors status

I have

- [ ] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [ ] Followed all necessary [style guidelines](https://climate-machine.github.io/CLIMA/latest/CodingConventions.html) and run `julia .dev/format.jl`
- [ ] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [ ] There are no open pull requests for this already
- [ ] CLIMA developers with relevant expertise have been assigned to review this submission
- [ ] The code conforms to the [style guidelines](https://climate-machine.github.io/CLIMA/latest/CodingConventions.html) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [ ] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

File: bors.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 status = [
-  "CLIMA",
+  "ClimateMachine.jl",
```
Comment (simonbyrne) [inline bors.toml:2]:
```suggestion
  "azure-ci",
```
https://dev.azure.com/climate-machine/ClimateMachine.jl/_build?definitionId=13&_a=summary

**Comment by charleskawczynski** [comment]
Going to fix a couple things

File: bors.toml (hunk 1 above)
Comment (charleskawczynski) [reply bors.toml]: Done.

**Pull request merged**